### PR TITLE
Update wrr_traffic_test.go

### PR DESCRIPTION
**Pull request by saarvind6481**
Cisco-Dp-1.9 pull request

### CODE REVIEW

**Review by sezhang2 — DISMISSED** [review]
I am still working on this experimental code. Please wait.

File: feature/experimental/qos/ate_tests/wrr_traffic_test/wrr_traffic_test.go [hunk 1]
```diff
@@ -349,15 +353,15 @@ func TestWrrTraffic(t *testing.T) {
 		"intf1-af3": {
 			frameSize:             1000,
 			trafficRate:           40,
-			expectedThroughputPct: 100.0,
+			expectedThroughputPct: 83.48,
```
Comment (sezhang2) [inline feature/experimental/qos/ate_tests/wrr_traffic_test/wrr_traffic_test.go:356]: Can you explain your calculation?

We may need to calculate this dynamically based on the weights.

**Comment by vbhan-cisco** [comment]
> Does the test pass in your lab with your change?

Yes we have tried these changes and they work

**Comment by guoshiuan** [comment]
b/270386083 has the internal test result.

**Review by guoshiuan — APPROVED** [review]
We (Sean, Viktor, Cisco team and I) had a discussion (b/268403096#comment9). Viktor has filed bugs to track the explanation of Cisco implementation, and we decided to move forward with this PR.

**Review by prinikasn — COMMENTED** [review]
/gcbrun

**Review by prinikasn — APPROVED** [review]

**Pull request merged**